hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -244,6 +244,21 @@
       stage: V2
       tests: [""]
     }
+    {
+      name: i2c_host_mode_toggle
+      desc: '''
+            Host mode: disable host mode during host mode sequence
+
+            Stimulus:
+              - Host sends an address and data but receives NAK response
+                since agent is reset before transaction is complete and
+                host mode is disabled
+            Checking:
+              - Check if DUT goes to Idle state after Host mode is disabled
```

Comment:
```suggestion
              - Check if DUT goes to Idle state after Host mode is disabled
              - Check that transactions process normally after recovery by running a smoketest vseq.
```

Reply:
Updated as per comments in [555310d](https://github.com/lowRISC/opentitan/compare/7c103bfd9522f53485c1b5385fc3f22e57bdcb2c..555310dc38f168b19cc1776a793e504c44b52d5b)